CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.6.6
+
+- Fixed octokit.issues undefined issue.
+
```

Comment:
it maybe more easy to understand the issue  if we can change change log like 'Fixed octokit name change to Octokit  caused undefined issue.